pkgs/development/perl-modules/Parse-Win32Registry/default.nix

```
@@ -0,0 +1,26 @@
+{ lib
+, perlPackages
+, fetchurl
+, perl
+}:
+
+perlPackages.buildPerlPackage rec {
+  pname = "Parse-Win32Registry";
+  version = "1.1";
+
+  src = fetchurl {
+    url = "mirror://cpan/authors/id/J/JM/JMACFARLA/Parse-Win32Registry-${version}.tar.gz";
+    hash = "sha256-wWOyAr5q17WPSEZJT/crjJqXloPKmU5DgOmsZWTcBbo=";
+  };
+
+  buildInputs = [ perl ];
+
+  meta = with lib; {
+    description = "Module for parsing Windows Registry files";
+    mainProgram = "Parse-Win32Registry";
+    homepage = "https://metacpan.org/pod/Parse::Win32Registry";
+    maintainers = with maintainers; [ d3vil0p3r ];
+    platforms = with platforms; linux ++ darwin;
+    license = with licenses; gpl1Plus;
```

Uses the [same Perl5 license,](https://metacpan.org/pod/Parse::Win32Registry#COPYRIGHT-AND-LICENSE) so 

```suggestion
    license = with licenses; [ artistic1 gpl1Plus ];
```